### PR TITLE
Docs for logging from Ruby

### PR DESCRIPTION
**Pull request by thijsc**
Basic docs for the first customers that will try logging from the Ruby integration.

### CODE REVIEW

**Comment by thijsc** [comment]
Nice 👍

**Comment by noccor** [comment]
@roytomeij - tone of voice check?

File: source/posts/logging/platforms/integrations/index.mdx [hunk 1]
```diff
@@ -0,0 +1,20 @@
...
+application is a good candidate, [send us an
+email](mailto:support@appsignal.com?subject=Logging%20beta)!
+
+</Notice>
+
+AppSignal supports logging directly from your AppSignal integration. The steps outlined in the [Configure Logging documentation](/logging/configuration.html) do not apply when setting up logging via your AppSignal integration.
+
+## Supported Integrations Setup
+
+Currently, logging is only available for our Ruby integration, logging will be expanded to support other integrations soon.
```
Comment (roytomeij) [inline source/posts/logging/platforms/integrations/index.mdx:18]:
```suggestion
Currently, logging is only available for our Ruby integration. Logging will be expanded to support other integrations soon.
```

File: source/posts/logging/platforms/integrations/index.mdx [hunk 2]
```diff
@@ -0,0 +1,20 @@
...
+
+- [Ruby](ruby.html)
```
Comment (roytomeij) [inline source/posts/logging/platforms/integrations/index.mdx:20]: This link returns a `404` in the preview.

File: source/posts/logging/configuration.mdx [hunk 3]
```diff
@@ -21,6 +21,13 @@ email](mailto:support@appsignal.com?subject=Logging%20beta)!
 
 ### Creating a Log Source
 
+<Notice variant="Notice">
+  💡 We are expanding support for Logging to AppSignal integrations. Currently,
+  we only support the Ruby integration. Integration logs do not need a log
+  source. Read more, in the [Logging From AppSignal
+  Integration](platforms/integrations).
```
Comment (roytomeij) [inline source/posts/logging/configuration.mdx:28]:
```suggestion
  Integrations](platforms/integrations).
```

File: source/posts/logging/configuration.mdx [hunk 4]
```diff
@@ -35,6 +42,9 @@ A pop-up will appear, prompting you to name your source. It's important to give
 
 We support sending logs to AppSignal from various sources and platforms, and are hard at work to add support for more.
 
+You can also send logs directly from supported AppSignal integrations,
+in this instance you don't need to create a source and should instead following seperate installation instructions for your integration. More information can be found in the [Logging From AppSignal Integration documentation](platforms/integrations).
```
Comment (roytomeij) [inline source/posts/logging/configuration.mdx:46]:
```suggestion
In this instance, you don't need to create a source and should follow separate installation instructions for your integration. More information can be found in the [Logging From AppSignal Integration documentation](platforms/integrations).
```

File: source/posts/logging/platforms/integrations/ruby.mdx [hunk 5]
```diff
@@ -0,0 +1,61 @@
...
+You can define custom attributes to send log information that can be used when filtering and querying logs;
+
+```ruby
+logger = Appsignal::Logger.new("invoice_helper")
+logger.info("Generating invoice for customer #{@customer.id}", { customer_id: @customer.id, date: DateTime.now })
+invoice = generate_invoice(@customer)
+logger.info("Generated invoice for customer #{@customer.id}", { customer_id: @customer.id, invoice_id: invoice.id})
+```
+
+You can query and filter on message contents, and attribute values from within the [Log Management](/logging/log-management.html) tool.
```
Comment (roytomeij) [inline source/posts/logging/platforms/integrations/ruby.mdx:61]:
```suggestion
You can query and filter on message contents and attribute values from within the [Log Management](/logging/log-management.html) tool.
```

**Review by roytomeij — CHANGES_REQUESTED** [review]

**Comment by noccor** [comment]
@roytomeij - feedback taken onboard, with some other minor adjustments + docs wide dead link check

File: source/posts/logging/configuration.mdx [hunk 6]
```diff
@@ -21,6 +21,13 @@ email](mailto:support@appsignal.com?subject=Logging%20beta)!
 
 ### Creating a Log Source
 
+<Notice variant="Notice">
+  💡 We are expanding support for Logging to AppSignal integrations. Currently,
+  we only support our Ruby integration. Integration logs do not need a log
+  source. Read more in the [Logging From AppSignal
+  Integrations](platforms/integrations) documentation.
```
Comment (noccor) [inline source/posts/logging/configuration.mdx:28]: Where my choice of capitalization bites, here it's AppSignal Integrations, referring to the page title.

**Comment by thijsc** [comment]
We are running a migration and will be ready to ship end of the day or tomorrow morning. Can we get the last details for this wrapped up?

**Review by roytomeij — APPROVED** [review]

**Comment by roytomeij** [comment]
Has the feature been shipped, so we can ship the docs as well?

**Pull request merged**